Added Checkpoints

getmikyled opened the pull request:
- Added the checkpoint script
- Added a blue capsule prefab for the checkpoint
- Added the material

getmikyled committed:
Added Checkpoints

Adds the script
Adds the prefab
Adds the blue material for the checkpoint prefab

getmikyled committed:
Fixed Material

TheStress reviewed (commented):
will close PR and wait for the `PlayerKillable` to be merged

getmikyled committed:
Merge branch 'develop' into checkpoints

getmikyled committed:
Updated Checkpoints Script

Fixed it so that when the player dies the player gets sent back to the position of the checkpoint

getmikyled committed:
Added CheckpointManger Prefab

getmikyled commented:
The checkpoints will not actually work until the function PlayerDied uses another way to respawn the player besides LoadScene

TheStress committed:
Merge branch 'develop' into checkpoints

TheStress committed:
checkpoints completed

- added player death resulting in respawn point (this is changed by checkpoints)
- checkpoint manager that contains all the checkpoints as children
- checkpoints that changes the current checkpoint based on sequence and sets player respawn point
- sequence of checkpoints are determined by child order in object hierarchy under manager
- added checkpoints to template scene

TheStress committed:
moved checkpoint prefabs for easier use
